doc/smart-switch/high-availability/smart-switch-ha-dpu-scope-dpu-driven-setup.md

```diff
@@ -159,6 +181,18 @@ Here are how the workflows look like for the typical planned operations:
 
 ### 8.1. HA set creation
 
+The HA bring-up workflow is described below -
+  - The DPU starts out with it's initial HA scope Role as Dead.
```

Comment:
nit: do you mind to remove the initial indention of the list?

Reply:
done